instant-xml-macros/src/se.rs

```diff
@@ -67,42 +70,79 @@ impl<'a> Serializer {
         ));
     }
 
-    pub fn process_named_field(
-        &mut self,
-        field: &syn::Field,
-        output: &'a mut TokenStream,
-        missing_prefixes: &'a mut BTreeSet<String>,
-    ) {
+    pub fn process_named_field(&mut self, field: &syn::Field) -> (TokenStream, bool) {
```

Comment:
We should pass in a `&mut TokenStream` instead of allocating a separate one in here.

Reply:
done